[EUI+] Add missing guidelines

weronikaolejniczak opened the pull request:
## Summary

This PR adds the missing guidelines for several components: `EuiEmptyPrompt`, `EuiToast`, `EuiTour`, Form controls and Selection controls.

## QA

Compare the content of all pages between [the old docs](https://eui.elastic.co/pr_8309/) and [the new docs](https://eui.elastic.co/pr_8309/new-docs):

- [x] [EuiEmptyPrompt’s Guidelines](https://eui.elastic.co/#/display/empty-prompt/guidelines) subpage
- [x] [EuiToast’s Guidelines](https://eui.elastic.co/#/display/toast/guidelines) subpage
- [x] [EuiTour’s Guidelines](https://eui.elastic.co/#/display/tour/guidelines) subpage
- [x] [Form controls’ Guidelines](https://eui.elastic.co/#/forms/form-controls/guidelines) subpage
- [x] [Selection controls’ Guidelines](https://eui.elastic.co/#/forms/selection-controls/guidelines) subpage

> [!NOTE]
> There are missing icons in `EuiEmptyPrompt` preview. It's been extracted to a separate task: https://github.com/elastic/eui/issues/8312 that will be tackled by @tkajtoch.

kibanamachine commented:
Preview staging links for this PR:
- Docs site: https://eui.elastic.co/pr_8309/
- Storybook: https://eui.elastic.co/pr_8309/storybook
- New docs: https://eui.elastic.co/pr_8309/new-docs

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414)
* Commit: acdfd9fcaabf1300760ea472146b51615b2ade6e

### Failed CI Steps
* [:hammer: EUI pull request test](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414#0194db35-5392-4ec0-a490-dd1b0ccaeef1)

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414","number":3414,"commit":"acdfd9fcaabf1300760ea472146b51615b2ade6e"}],"number":3414}
buildkite-pr-comment-->

kibanamachine commented:
Preview staging links for this PR:
- Docs site: https://eui.elastic.co/pr_8309/
- Storybook: https://eui.elastic.co/pr_8309/storybook
- New docs: https://eui.elastic.co/pr_8309/new-docs

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3419)
* Commit: 2bc584dc7ca564467c99e30d01ed420b8f78c0d0

### History
* :broken_heart: [Build #3414](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414) failed acdfd9fcaabf1300760ea472146b51615b2ade6e

cc @weronikaolejniczak

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3419","number":3419,"commit":"2bc584dc7ca564467c99e30d01ed420b8f78c0d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414","number":3414,"commit":"acdfd9fcaabf1300760ea472146b51615b2ade6e"}],"number":3419}
buildkite-pr-comment-->

acstll commented:
Here's what I found 🙂 

## [EuiEmptyPrompt’s Guidelines](https://eui.elastic.co/#/display/empty-prompt/guidelines) subpage

- in the "Empty state types, goals, and recommendations" section, some styles like the sidebar look different, also the template radio buttons seem overflowed [screenshot 1]
- image comparisons don't have the same height, and in some cases the blueish background is missing (not sure if intended) [screenshot 2]
- [not part of the changes] the "empty prompt usage guidelines" link in the Overview page that links to the Guidelines page is broken

[1]

![Screenshot 2025-02-10 at 12 11 57](https://github.com/user-attachments/assets/b479b581-ee70-4c1a-a0a3-5d42b726f675)

[2]

| Old | New |
|----|----|
| ![Screenshot 2025-02-10 at 12 47 06 old](https://github.com/user-attachments/assets/05b8ab53-59a1-49e2-8e48-186244c3cbb8) |  ![Screenshot 2025-02-10 at 12 47 18 new](https://github.com/user-attachments/assets/626ae9b1-77b0-4ae0-b632-83be1e507534) |

## [EuiToast’s Guidelines](https://eui.elastic.co/#/display/toast/guidelines) subpage

- in the examples, the "danger" button has a larger font size in all instances [screenshot 3]
- [not part of the changes] the "toast usage guidelines" link in the Overview page that links to the Guidelines page is broken

[3]

| Old | New |
|----|----|
| ![Screenshot 2025-02-10 at 12 17 28 old](https://github.com/user-attachments/assets/13cb66bd-e4b5-4bb4-aad4-e1014ab0e58e) | ![Screenshot 2025-02-10 at 12 17 24 new](https://github.com/user-attachments/assets/3cc391d7-7315-466a-8773-3a49618621fd) |

## [EuiTour’s Guidelines](https://eui.elastic.co/#/display/tour/guidelines) subpage

No comments 👍 

## [Form controls’ Guidelines](https://eui.elastic.co/#/forms/form-controls/guidelines) subpage

- the comparison do/dont snippets don't have the same height, maybe it's intended but I think it looks better on the old site
- in "Hint text" and "Placeholder text/In fields" sections there's a `dont` that should be `do` [screenshot 4]

[4]

| Old | New |
|----|----|
| ![Screenshot 2025-02-10 at 12 24 36 old](https://github.com/user-attachments/assets/c45c8f19-6220-4d93-a532-98c92271eb45) | ![Screenshot 2025-02-10 at 12 24 39 new](https://github.com/user-attachments/assets/51a88f28-bb02-45fd-b7e7-0ac53fc4cf46) |

## [Selection controls’ Guidelines](https://eui.elastic.co/#/forms/selection-controls/guidelines) subpage

- styles in the "Switch labels" do/dont section are strange [screenshot 5]

[5]

| Old | New |
|----|----|
| ![Screenshot 2025-02-10 at 12 27 53 old](https://github.com/user-attachments/assets/a5b47385-04f4-44ea-8375-8330dc6259b2) | ![Screenshot 2025-02-10 at 12 28 00 new](https://github.com/user-attachments/assets/32ca798f-62f8-409d-abc1-2802a8d32ff5) |

acstll reviewed (changes requested):
Thank you @weronikaolejniczak for tackling this one!

The code looks perfect to me, but I found a few things looking at it from a user/designer perspective, I left them in [this comment](https://github.com/elastic/eui/pull/8309#issuecomment-2647878883). Maybe some changes are intended, let me know.

weronikaolejniczak commented:
I fixed whatever I could, some things I couldn't without going outside of the scope of this PR and I'd propose to fix them on another task 😄 

### EuiEmptyPrompt ❌ 

This one... 💀 With a bit of styling I managed to fix the "image radios". The preview I was not able to get it to work as expected, hence https://github.com/elastic/eui/issues/8312. Do you have an idea how to fix it?

I updated the link 👍🏻 

https://github.com/user-attachments/assets/076a6859-9392-4f40-8190-e5d55a70a17a

### EuiToast ❌ 

This one is super weird! 🤔 The danger buttons do have bigger font-size although the same `size` is passed. If you take a look at the [Button](https://eui.elastic.co/pr_8309/new-docs/docs/navigation/button/) page, it displays as expected there.

There are no additional styles passed to them. Something must be influencing it from higher up? Do you have an idea what could be wrong?

I updated the link 👍🏻 

<details><summary>Screenshots</summary>
<p>
![Screenshot 2025-02-12 at 15 07 26](https://github.com/user-attachments/assets/99e9a11f-a44f-44e5-8ead-a54f7a65ab0d)
![Screenshot 2025-02-12 at 15 07 39](https://github.com/user-attachments/assets/c169aabc-25f8-4c5d-ab86-953017270fcd)
![Screenshot 2025-02-12 at 15 11 17](https://github.com/user-attachments/assets/18d63407-b8ec-47ca-b12b-f7f0e06252b2)
</p>
</details> 

### Form controls ✅ 

I made the height of guideline panels equal and I changed some (incorrect) "don't" sections to "do":

<details><summary>Screenshots</summary>
<p>
![Screenshot 2025-02-12 at 15 00 26](https://github.com/user-attachments/assets/e05ee1aa-b4dc-407b-8a9f-1ea7dfe286d6)
![Screenshot 2025-02-12 at 15 00 41](https://github.com/user-attachments/assets/2a430fe9-4823-48fa-bcc9-b898dbaa7e19)
![Screenshot 2025-02-12 at 15 00 46](https://github.com/user-attachments/assets/9a9e9ba5-3caf-4a3a-ab00-f04654e5dd1b)
</p>
</details> 

### Selection controls ✅ 

I agree this table looks weird 👍🏻 What I found is that this is the responsive behavior of `EuiBasicTable`. The width of the page content is smaller than in the original docs, causing the table to pass a breakpoint and display like this. When I applied `responsiveBreakpoint={false}` it looks as in the original docs. I'm sure it's fine to get rid of the responsive behavior because it doesn't serve the purpose of this guideline specifically.

<details><summary>Screenshots</summary>
<p>
![Screenshot 2025-02-12 at 14 05 58](https://github.com/user-attachments/assets/ff70d9a8-bb3b-4ec9-bcae-e8cf5ce080e1)
</p>
</details> 

The code in: `packages/website/docs/components/forms/selection_controls/guidelines.mdx` is the same as in: `packages/eui/src-docs/src/views/selection_controls/guidelines.js`.

kibanamachine commented:
Preview staging links for this PR:
- Docs site: https://eui.elastic.co/pr_8309/
- Storybook: https://eui.elastic.co/pr_8309/storybook
- New docs: https://eui.elastic.co/pr_8309/new-docs

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3434)
* Commit: 5b34e638973a4bb57b78a36e559d5ae7fb559284

### Failed CI Steps
* [:hammer: EUI pull request test](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3434#0194fa94-59fd-4346-a520-91d9433896cf)

### History
* :green_heart: [Build #3419](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3419) succeeded 2bc584dc7ca564467c99e30d01ed420b8f78c0d0
* :broken_heart: [Build #3414](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414) failed acdfd9fcaabf1300760ea472146b51615b2ade6e

cc @weronikaolejniczak

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3434","number":3434,"commit":"5b34e638973a4bb57b78a36e559d5ae7fb559284"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3419","number":3419,"commit":"2bc584dc7ca564467c99e30d01ed420b8f78c0d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414","number":3414,"commit":"acdfd9fcaabf1300760ea472146b51615b2ade6e"}],"number":3434}
buildkite-pr-comment-->

kibanamachine commented:
Preview staging links for this PR:
- Docs site: https://eui.elastic.co/pr_8309/
- Storybook: https://eui.elastic.co/pr_8309/storybook
- New docs: https://eui.elastic.co/pr_8309/new-docs

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3435)
* Commit: 208b739f1efd5d79bb30877a86b566454f0ddfaf

### History
* :broken_heart: [Build #3434](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3434) failed 5b34e638973a4bb57b78a36e559d5ae7fb559284
* :green_heart: [Build #3419](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3419) succeeded 2bc584dc7ca564467c99e30d01ed420b8f78c0d0
* :broken_heart: [Build #3414](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414) failed acdfd9fcaabf1300760ea472146b51615b2ade6e

cc @weronikaolejniczak

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3435","number":3435,"commit":"208b739f1efd5d79bb30877a86b566454f0ddfaf"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3434","number":3434,"commit":"5b34e638973a4bb57b78a36e559d5ae7fb559284"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3419","number":3419,"commit":"2bc584dc7ca564467c99e30d01ed420b8f78c0d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414","number":3414,"commit":"acdfd9fcaabf1300760ea472146b51615b2ade6e"}],"number":3435}
buildkite-pr-comment-->

acstll commented:
> I fixed whatever I could, some things I couldn't without going outside of the scope of this PR and I'd propose to fix them on another task 😄

@weronikaolejniczak I agree we could fix these later on 👍 

I'll take a quick look first at your comments for EuiEmptyPrompt and EuiToast first, and then re-run this locally for a final check 🤓 

(btw the screenshots inside the <details> toggles are not rendering… maybe there's a space missing somewhere? putting markdown inside html does not always work, but I'm not sure that's the case…)

acstll commented on packages/website/docs/components/display/empty_prompt/guidelines.mdx:
```diff
@@ -0,0 +1,101 @@
+Use the horizontal layout when you have a long description, multiple calls to action, and a footer. For this type of layout, an illustration is required.
+
+<EuiFlexGroup gutterSize="m">
+  <Guideline text="Use the vertical layout when the text is 1 to 2 sentences.">
```
I think it's worth making these pairs look like in the old site, same height and background color… but it seems this could be fixed globally in the `Guideline` component, so something we could fix in a different PR as you suggest?

**Old**
![Screenshot 2025-02-10 at 12 47 06 old](https://github.com/user-attachments/assets/d288074f-b049-449e-a0b1-737fc7a79006)

**New**
![Screenshot 2025-02-10 at 12 47 18 new](https://github.com/user-attachments/assets/a88a3578-44a2-4753-846a-622e6e6ba3b8)

weronikaolejniczak replied:
It's a great thought to fix it globally on a separate PR 👍🏻 Because it's not a blocker, I created a task for it in M2: [#8330](https://github.com/elastic/eui/issues/8330)

I can easily fix this case locally 😄

weronikaolejniczak replied:
Fixed on f67b4d3

acstll replied:
oh, thank you! maybe you can also fix it then for the other sections in the same page? 🙏  (Icons and illustrations and Learn more links)

weronikaolejniczak replied:
@acstll done on 7808a49

weronikaolejniczak replied:
the fact that those guidelines are not of the same height is not a dealbreaker to me - definitely, it looks better when they are but we might as well go the easy route for now and improve in M2, so I agree 100% 😄 reverting...

weronikaolejniczak replied:
done ✅

acstll commented on packages/website/docs/components/display/empty_prompt/guidelines.mdx:
```diff
@@ -0,0 +1,101 @@
+
+import { TypesOfEmptyStates } from './types_of_empty_states';
+
+<TypesOfEmptyStates />
```
Is something we could fix in this PR? I know there's #8312 but I the Sidebar snippet has a layout issue… This looks specific to this one component guidelines. Wdyt?

**Old**
![17-63fd8988a695c10f84299fd227bd6d1d](https://github.com/user-attachments/assets/b5e1f949-6c08-4f73-8390-96c79701c86c)

**New**
![17-7259661f8d7f981a65ec1260b456d6e9](https://github.com/user-attachments/assets/b894185c-374c-43f9-aba6-4c255bedc9fe)

weronikaolejniczak replied:
Yes, 100% agreed, we can fix it here. I remember struggling with this and when I sat down to it quickly now I couldn't come up with any solution. The way the code snippet component works is really magical to me 🪄 

I'll try to dig deeper and ping in case of doubts!

weronikaolejniczak replied:
After alignment with @acstll, we decided to move this issue to https://github.com/elastic/eui/issues/8164 as it seems the cause is in how `<Demo>` behaves with `<EuiPageTemplate>` component.

acstll reviewed (changes requested):
1 (or 2 small things) more 🙏 🙏 🙏

kibanamachine commented:
Preview staging links for this PR:
- Docs site: https://eui.elastic.co/pr_8309/
- Storybook: https://eui.elastic.co/pr_8309/storybook
- New docs: https://eui.elastic.co/pr_8309/new-docs

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3455)
* Commit: f67b4d3fef2acd074fe217b99b097b5dad922337

### History
* :green_heart: [Build #3435](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3435) succeeded 208b739f1efd5d79bb30877a86b566454f0ddfaf
* :broken_heart: [Build #3434](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3434) failed 5b34e638973a4bb57b78a36e559d5ae7fb559284
* :green_heart: [Build #3419](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3419) succeeded 2bc584dc7ca564467c99e30d01ed420b8f78c0d0
* :broken_heart: [Build #3414](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414) failed acdfd9fcaabf1300760ea472146b51615b2ade6e

cc @weronikaolejniczak

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3455","number":3455,"commit":"f67b4d3fef2acd074fe217b99b097b5dad922337"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3435","number":3435,"commit":"208b739f1efd5d79bb30877a86b566454f0ddfaf"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3434","number":3434,"commit":"5b34e638973a4bb57b78a36e559d5ae7fb559284"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3419","number":3419,"commit":"2bc584dc7ca564467c99e30d01ed420b8f78c0d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414","number":3414,"commit":"acdfd9fcaabf1300760ea472146b51615b2ade6e"}],"number":3455}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3459)
* Commit: 7808a49b4affa84e6884f164cc39d11d1ce59e89

### Failed CI Steps
* [:hammer: EUI pull request test](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3459#01951486-dedd-484c-9d1c-10ee46804fce)

### History
* :green_heart: [Build #3455](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3455) succeeded f67b4d3fef2acd074fe217b99b097b5dad922337
* :green_heart: [Build #3435](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3435) succeeded 208b739f1efd5d79bb30877a86b566454f0ddfaf
* :broken_heart: [Build #3434](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3434) failed 5b34e638973a4bb57b78a36e559d5ae7fb559284
* :green_heart: [Build #3419](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3419) succeeded 2bc584dc7ca564467c99e30d01ed420b8f78c0d0
* :broken_heart: [Build #3414](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414) failed acdfd9fcaabf1300760ea472146b51615b2ade6e

cc @weronikaolejniczak

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3459","number":3459,"commit":"7808a49b4affa84e6884f164cc39d11d1ce59e89"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3455","number":3455,"commit":"f67b4d3fef2acd074fe217b99b097b5dad922337"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3435","number":3435,"commit":"208b739f1efd5d79bb30877a86b566454f0ddfaf"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3434","number":3434,"commit":"5b34e638973a4bb57b78a36e559d5ae7fb559284"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3419","number":3419,"commit":"2bc584dc7ca564467c99e30d01ed420b8f78c0d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3414","number":3414,"commit":"acdfd9fcaabf1300760ea472146b51615b2ade6e"}],"number":3459}
buildkite-pr-comment-->

kibanamachine commented:
Preview staging links for this PR:
- Docs site: https://eui.elastic.co/pr_8309/
- Storybook: https://eui.elastic.co/pr_8309/storybook
- New docs: https://eui.elastic.co/pr_8309/new-docs

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3471)
* Commit: 8815898b82cbd7410de43e401902753eac6c293b

### History
* :green_heart: [Build #3459](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3459) succeeded 7808a49b4affa84e6884f164cc39d11d1ce59e89
* :green_heart: [Build #3455](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3455) succeeded f67b4d3fef2acd074fe217b99b097b5dad922337
* :green_heart: [Build #3435](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3435) succeeded 208b739f1efd5d79bb30877a86b566454f0ddfaf
* :broken_heart: [Build #3434](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3434) failed 5b34e638973a4bb57b78a36e559d5ae7fb559284
* :green_heart: [Build #3419](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3419) succeeded 2bc584dc7ca564467c99e30d01ed420b8f78c0d0

cc @weronikaolejniczak

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3471","number":3471,"commit":"8815898b82cbd7410de43e401902753eac6c293b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3459","number":3459,"commit":"7808a49b4affa84e6884f164cc39d11d1ce59e89"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3455","number":3455,"commit":"f67b4d3fef2acd074fe217b99b097b5dad922337"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3435","number":3435,"commit":"208b739f1efd5d79bb30877a86b566454f0ddfaf"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3434","number":3434,"commit":"5b34e638973a4bb57b78a36e559d5ae7fb559284"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/3419","number":3419,"commit":"2bc584dc7ca564467c99e30d01ed420b8f78c0d0"}],"number":3471}
buildkite-pr-comment-->

acstll reviewed (approved):
thank you very much @weronikaolejniczak 🙏
